node/src/BaseClient.ts

```diff
@@ -3494,6 +3495,36 @@ export class BaseClient {
         return this.createWritePromise(createObjectRefcount(key));
     }
 
+    /**
+     * Copies the value stored at the `source` to the `destination` key if the `destination` key does not
+     * yet exist.
```

Comment:
Can we follow the python docs here? The current doc suggests that this command can only be applied if destination doesn't exist, but the command can still be applied if it does exist by passing the replace arg.
```suggestion
     * Copies the value stored at the `source` to the `destination` key. When `replace` is `true`,
     * removes the `destination` key first if it already exists, otherwise performs no action.
```